src/pages/admin/create-notes.tsx

```diff
@@ -0,0 +1,418 @@
+import React from 'react';
+import { EditorState, ContentState, convertToRaw } from 'draft-js';
+import { Editor } from 'react-draft-wysiwyg'
+import Amplify from 'aws-amplify';
+import AdminMenu from '../../components/Menu/AdminMenu';
+import BlogPreview from './BlogPreview';
+import { Authenticator, SignOut, Greetings } from 'aws-amplify-react';
+import awsmobile from '../../aws-exports';
+import * as queries from '../../graphql/queries';
+import * as mutations from '../../graphql/mutations';
+import { GRAPHQL_AUTH_MODE } from '@aws-amplify/api/lib/types';
+import { API } from 'aws-amplify';
+import { Storage } from 'aws-amplify';
+import { Modal } from 'reactstrap';
+import { v1 as uuidv1 } from 'uuid';
+import draftToHtml from 'draftjs-to-html';
+import htmlToDraft from 'html-to-draftjs';
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import format from 'date-fns/format';
+import parse from 'date-fns/parse';
+import getDay from 'date-fns/getDay';
+import './create-notes.scss';
+
+Amplify.configure(awsmobile);
+const federated = {
+    google_client_id: '',
+    facebook_app_id: '579712102531269',
+    amazon_client_id: ''
+};
+
+const Index = () => (
+    <div>
+        <Authenticator federated={federated} hide={[Greetings, SignOut]}>
+            <AuthIndexApp></AuthIndexApp>
+        </Authenticator>
+    </div>
+)
+interface Props {
+    authState?: any
+}
+interface State {
+  editorState: any
+  showPreview: boolean
+  notesList: any
+  selectedTags: any
+  noteObject: any
+  noteEditObject: any
+  showAlert: string
+  currentTag: string
+  moreOptions: boolean
+  showEditModal: boolean
+  delete: string
+  understand: string
+  noteEdit: any
+  date: any
+  pdf: string
+  dateWarning: string
+  title: string
+}
+
+class AuthIndexApp extends React.Component<Props, State> {
+
+  render() {
+      if (this.props.authState === "signedIn") {
+          return (
+              <div>
+                  <IndexApp></IndexApp>
+              </div>
+          );
+      } else {
```

Comment:
You don't need an else if the true branch returns.

Reply:
Error: AuthIndexApp(...): Nothing was returned from render. This usually means a return statement is missing. Or, to render nothing, return null.